### PR TITLE
openvpn: T5634: Remove support for insecure DES and Blowfish ciphers

### PR DESCRIPTION
**Pull request by dmbaturin**
## Change Summary

Remove DES and Blowfish ciphers from OpenVPN configs if they are set explicitly to allow removal of the legacy crypto provider. Configs with those options unset allow clients to negotiate any of the support ciphers.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [x] Other (please describe): Legacy functionality removal

## Related Task(s)

* https://vyos.dev/T5634

## Component(s) name

OpenVPN.

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/migration-scripts/openvpn/0-to-1 [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        cipher_path = ['interfaces', 'openvpn', i, 'encryption', 'cipher']
+        if config.exists(cipher_path):
+            cipher = config.return_value(cipher_path)
+            if cipher in ['des', 'bf128', 'bf256']:
+                config.delete(cipher_path)
+
+        ncp_cipher_path = ['interfaces', 'openvpn', i, 'encryption', 'ncp-ciphers']
+        if config.exists(ncp_cipher_path):
+            ncp_ciphers = config.return_values(['interfaces', 'openvpn', i, 'encryption', 'ncp-ciphers'])
+            print(ncp_ciphers)
```
Comment (sever-sever) [inline src/migration-scripts/openvpn/0-to-1]: Print is not required

**Review by sever-sever — CHANGES_REQUESTED** [review]

File: src/migration-scripts/openvpn/0-to-1 (hunk 1 above)
Comment (dmbaturin) [reply src/migration-scripts/openvpn/0-to-1]: Good point, that's a stray debug print. Removed.

**Review by sever-sever — APPROVED** [review]

**Review by sarthurdev — APPROVED** [review]

**Review by c-po — CHANGES_REQUESTED** [review]
Please also create a mew component version file for OpenVPN here: https://github.com/vyos/vyos-1x/tree/current/interface-definitions/include/version

**Comment by dmbaturin** [comment]
> Please also create a mew component version file for OpenVPN here: https://github.com/vyos/vyos-1x/tree/current/interface-definitions/include/version

Damn, the file didn't get included in the commit. I've fixed it now.

**Review by jestabro — APPROVED** [review]

**Pull request merged**